lib/Doctrine/ORM/Mapping/Index.php

```diff
@@ -31,26 +31,26 @@
 #[Attribute(Attribute::TARGET_CLASS | Attribute::IS_REPEATABLE)]
 final class Index implements Annotation
 {
-    /** @var string */
+    /** @var string|null */
     public $name;
 
-    /** @var array<string> */
+    /** @var array<string>|null */
```

Comment:
This might be out of scope, but shouldn't use generic array declaration with only one type parameter.

```suggestion
    /** @var array<array-key,string>|null */
```
or
```suggestion
    /** @var string[]|null */
```

Reply:
That's why I created #9026.